projects/components/src/table/table.component.ts

```diff
@@ -426,13 +425,6 @@ export class TableComponent
     }
   }
 
-  public ngAfterViewInit(): void {
```

Comment:
@itssharmasandeep did you test this for all forms of the table? Normal, expanded, and waterfall?